Jenkinsfile

```diff
@@ -4,8 +4,42 @@ properties(
   ]
 )
 
+// https://issues.jenkins-ci.org/browse/JENKINS-33511
+def set_workspace() {
+  if(env.WORKSPACE == null) {
+    env.WORKSPACE = pwd()
+  }
+}
+
 node('buildvm-devops') {
-  sshagent(['openshift-bot']) { // git repo privileges stored in Jenkins credential store
-    echo "Bruno's branch split pipeline"
+  try {
+    deleteDir()
+    set_workspace()
+    stage('clone') {
+      dir('aos-cd-jobs') {
+        checkout scm
+        sh 'git checkout master'
+      }
+    }
+    stage('run') {
+      sshagent(['openshift-bot']) { // git repo privileges stored in Jenkins credential store
+        sh '''\
+virtualenv env/
+env/bin/pip install gitpython
+export PYTHONPATH=$PWD/aos-cd-jobs
+env/bin/python aos-cd-jobs/aos_cd_jobs/pruner.py
```

Comment:
This is forcing the interpreter to be the one inside the virtualenv.  To
execute the scripts directly, we'd have to do:

```sh
source env/bin/activate
aos-cd-jobs/aos_cd_jobs/pruner.py
aos-cd-jobs/aos_cd_jobs/updater.py
```